src/plugins/data/common/search/tabify/response_writer.ts

```diff
@@ -28,19 +29,18 @@ export class TabbedAggResponseWriter {
   metricBuffer: BufferColumn[] = [];
 
   private readonly partialRows: boolean;
+  private readonly params: Partial<TabbedResponseWriterOptions>;
```

Comment:
Can `params` be called something else that is more meaningful?

Reply:
open for suggestions, but as this it not our public API i wouldn't spend too much time on it.